Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingAccelerationStructurePass.cpp

```diff
@@ -47,7 +47,76 @@ namespace AZ
 
         void RayTracingAccelerationStructurePass::FrameBeginInternal(FramePrepareParams params)
         {
+            m_timestampResult = AZ::RPI::TimestampResult();
+            if(GetScopeId().IsEmpty())
+            {
+                InitScope(AZ::RHI::ScopeId(GetPathName()), AZ::RHI::HardwareQueueClass::Graphics);
+            }
+
             params.m_frameGraphBuilder->ImportScopeProducer(*this);
+
+            ReadbackScopeQueryResults();
+        }
+
+        AZ::RHI::Ptr<AZ::RPI::Query> RayTracingAccelerationStructurePass::GetQuery(AZ::RPI::ScopeQueryType queryType)
+        {
+            auto typeIndex{ static_cast<uint32_t>(queryType) };
+            if (!m_scopeQueries[typeIndex])
+            {
+                AZ::RHI::Ptr<AZ::RPI::Query> query;
```

Comment:
You don't need AZ::

Reply:
That makes sense. I have removed AZ:: from the remaining code introduced in this PR